web/controllers/registration_controller.ex

```diff
@@ -0,0 +1,26 @@
+defmodule PhoenixTwitter.RegistrationController do
+  use PhoenixTwitter.Web, :controller
+  alias PhoenixTwitter.User
+
+  plug :scrub_params, "user" when action in [:create]
+  
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
+
+    render conn, "new.html", changeset: changeset
+  end
+
+  def create(conn, %{"user" => user_params}) do
+    changeset = User.changeset(%User{}, user_params)
+
+    case PhoenixTwitter.Registration.create(changeset, PhoenixTwitter.Repo) do
```

Comment:
Pwede naman to. Consolidated lang lahat sa isang module yung authentication-shiz mo and mas `phoenix-feels` kapag nasa plug :D